src/libcore/array.rs

```diff
@@ -124,14 +124,6 @@ macro_rules! array_impls {
             }
 
             #[stable(feature = "rust1", since = "1.0.0")]
-            #[cfg(stage0)]
-            impl<T:Copy> Clone for [T; $N] {
-                fn clone(&self) -> [T; $N] {
-                    *self
-                }
-            }
```

Comment:
Oh. This is going to make the reversal at #44045 more difficult (if the decision turns out to be "not close").